src/commands/build.ts

```diff
@@ -85,36 +83,58 @@ export class BuildCommand implements Command {
     },
   ];
 
-  run(options: CommandOptions, config: ProjectConfig): Promise<any> {
+  async run(options: CommandOptions, config: ProjectConfig): Promise<any> {
     // Defer dependency loading until this specific command is run
-    const build = require('../build/build').build;
+    let build = require('../build/build').build;
+    const hasCLIArgumentsPassed = this.args.some(arg => {
+      return typeof options[arg.name] !== 'undefined';
```

Comment:
Because if someone has redefined `undefined` then all hell will break loose, and we will find and ~~prosecute the offenders to the fullest extent of the law~~ remove that dependency. We can't possibly avoid the `undefined` reference in entire whole app plus dependencies, so I'd rather do the simple, sanity-assuming thing. You could also check against `null` or use the `in` operator, both of which in this case will be effectively the same.